modules/howtos/pages/observability-metrics.adoc

```diff
@@ -20,7 +20,7 @@ Metrics broadly fall into the following categories:
 Right now only the first category is implemented by the SDK, more are planned.
 
 
-== The Default AggregatingMeter
+== The Default LoggingMeter
 
 The default implementation aggregates and logs request and response metrics.
```

Comment:
Should we mention that it has a different name since 3.1?

Reply:
Good shout, I'll add a note

Reply:
Thanks  :+1: